sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubProducerClient/CreateBatchOptionsTests.cs

```diff
@@ -38,6 +38,29 @@ public void CloneProducesACopy()
             Assert.That(clone.MaximumSizeInBytes, Is.EqualTo(options.MaximumSizeInBytes), "The maximum size should match.");
         }
 
+        /// <summary>
+        ///   Verifies functionality of the <see cref="CreateBatchOptions.ToSendOptions" />
+        ///   method.
+        /// </summary>
+        ///
+        [Test]
+        public void ToSendOptionsTranslatesProperly()
+        {
+            var options = new CreateBatchOptions
+            {
+                PartitionId = "0",
+                PartitionKey = "some_partition_123",
+                MaximumSizeInBytes = (int.MaxValue + 122L)
+            };
+
+            var sendOptions = options.ToSendOptions();
+            Assert.That(sendOptions, Is.TypeOf<SendOptions>(), "The send options should be a SendOptions instance.");
+            Assert.That(sendOptions, Is.Not.Null, "The send options should not be null.");
```

Comment:
Maybe I'm just overthinking, but I think it makes more sense to invert these lines. I think the second assertion would never be false if the first one passed.

Reply:
It made more sense when the batch options was inherited.   I flopped them.